txsni/snimap.py

```diff
@@ -170,12 +234,13 @@ class HostDirectoryMap(object):
     def __init__(self, directoryPath):
         self.directoryPath = directoryPath
 
-
     def __getitem__(self, hostname):
         if hostname is None:
-            hostname = "DEFAULT"
+            hostname = b"DEFAULT"
```

Comment:
In the original, hostname was either bytes or string; that meant Python 3 would only format KeyError() for DEFAULT and not for sni hostnames.

Reply:
This sounds like a bug that should be fixed separately.